site/content/docs/5.1/components/scrollspy.md

```diff
@@ -244,18 +252,63 @@ Scrollspy also works with `.list-group`s. Scroll the area next to the list group
 </div>
 ```
 
-## Usage
+## Example with simple anchors
 
-### Via data attributes
+Scrollspy is not limited to nav components and list groups, but will work on any `<a>` anchor elements in the current document. Scroll the area and watch the `.active` class change.
 
-To easily add scrollspy behavior to your topbar navigation, add `data-bs-spy="scroll"` to the element you want to spy on (most typically this would be the `<body>`). Then add the `data-bs-target` attribute with the ID or class of the parent element of any Boosted `.nav` component.
+<div class="bd-example">
+  <div class="row">
+    <div class="col-4">
+      <div id="simple-list-example" class="d-flex flex-column simple-list-example-scrollspy text-center">
+        <a href="#simple-list-item-1">Item 1</a>
+        <a href="#simple-list-item-2">Item 2</a>
+        <a href="#simple-list-item-3">Item 3</a>
+        <a href="#simple-list-item-4">Item 4</a>
+        <a href="#simple-list-item-5">Item 5</a>
+      </div>
+    </div>
+    <div class="col-8">
+      <div data-bs-spy="scroll" data-bs-target="#simple-list-example" data-bs-offset="0" data-bs-smooth-scroll="true" class="scrollspy-example" tabindex="0">
+        <h4 id="simple-list-item-1">Item 1</h4>
```

Comment:
Do we need to add some changes everywhere with a surrounding `<div>` ?

Reply:
I haven't looked at the new code for the scrollspy but I've found it weird; a different HTML structure for this use case.